lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -0,0 +1,111 @@
+#!powershell
+# This file is part of Ansible
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Author: Erwan Quelin (mail:erwan.quelin@gmail.com - GitHub: @equelin - Twitter: @erwanquelin)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = 'Stop'
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
+
+# Modules parameters
+
+$src = Get-AnsibleParam -obj $params -name "src" -type "string" -failifempty $true
+$version = Get-AnsibleParam -obj $params -name "version" -type "string" -failifempty $false
```

Comment:
So I think this is confusing, can you please rename this parameter to `minimum_version`and/or `min_version`. And also change the internal name to reflect this. More readable code.

Reply:
Ok for `minimum_version`